.circleci/config.yml

```diff
@@ -278,15 +267,11 @@ jobs:
       - run:
           name: Build dependencies
           command: |
-            make get_tools get_vendor_deps
+            make get_tools
       - persist_to_workspace:
           root: .
           paths:
             - "release-version.source"
-      - save_cache:
-          key: v1-release-deps-{{ .Branch }}-{{ .Revision }}
-          paths:
-            - "vendor"
```

Comment:
Please, do not delete the parts that enable a cache (but make it use modules too)!  

There is still a 
```yml
- restore_cache:
          keys:
            - v1-release-deps-{{ .Branch }}-{{ .Revision }}
```
left below. If there are any good reasons to not use a cache, then we'd need to delete those lines too.